showcase/plot/line-chart-gl.js

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
We've actually run into this issue before! i think @ibgreen suggested that we just remove the copyright and have a single copy right page at the root of the library, which would probably be better